src/v2/Apps/Auction/AuctionApp.tsx

```diff
@@ -113,9 +113,7 @@ export const AuctionAppFragmentContainer = createFragmentContainer(AuctionApp, {
 
       internalID
       coverImage {
-        cropped(width: 1800, height: 600, version: "normalized") {
-          src
-        }
+        url
```

Comment:
This should be fine but I'd probably figure out what versions this image has and then specify 2 or 3 in size order

Reply:
fixed 👍